src/pages/LoginPage.jsx

```diff
@@ -9,6 +10,8 @@ export const LoginPage = ({ currentUser, setCurrentUser }) => {
   const [isLoading, setIsLoading] = useState(false);
   const [isInvalid, setIsInvalid] = useState(false);
 
+  const navigate = useNavigate();
+
   useEffect(() => {
     setIsLoading(true);
     listUsers().then((users) => {
```

Comment:
Having a user type in a username can be troublesome. I would just give them a list of users that they can select one from.